packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -50,8 +50,9 @@ void UpdateRootViewSizeToAppWindow(
   // Do not relayout when minimized
   if (window.Presenter().as<winrt::Microsoft::UI::Windowing::OverlappedPresenter>().State() !=
       winrt::Microsoft::UI::Windowing::OverlappedPresenterState::Minimized) {
-    rootView.Arrange(size);
-    rootView.Size(size);
+    winrt::Microsoft::ReactNative::LayoutConstraints constraints;
+    constraints.MaximumSize = constraints.MinimumSize = size;
+    rootView.Arrange(constraints, {0, 0});
```

Comment:
do we expect the viewportoffset will always be {0,0}?

Reply:
No.  In cases where the rootview is not taking up the entire content of the host, or in particular is not positioned at 0,0 within the host, then the viewportOffset would be non-zero.